digdag-core/src/main/java/io/digdag/core/database/migrate/Migration_20200716114008_AddLastAttemptIdIndexToSessions.java

```diff
@@ -0,0 +1,23 @@
+package io.digdag.core.database.migrate;
+
+import org.skife.jdbi.v2.Handle;
+
+public class Migration_20200716114008_AddLastAttemptIdIndexToSessions
+    implements Migration
+{
+    @Override
+    public void migrate(Handle handle, MigrationContext context)
+    {
+        if (context.isPostgres()) {
+            handle.update("create index concurrently sessions_on_last_attempt_id on sessions (last_attempt_id)");
+        } else {
+            handle.update("create index sessions_on_last_attempt_id on sessions (last_attempt_id)");
+        }
+    }
+
+    @Override
+    public boolean noTransaction(MigrationContext context)
+    {
+        return context.isPostgres();
```

Comment:
This is needed because `create index concurrently` fails in a transaction?

Reply:
Yes, it's exactly so.